cli/src/cpp/utils/endpoint_subelement_parser.cpp

```diff
@@ -43,91 +43,98 @@ void endpoint_subelement_parser(
     message << ((endpoint == DDSEntity::DATAREADER) ? "DataReader <" : "DataWriter <") << element << ">";
 
     bool print_usage = false;
-    print_usage = subelement.empty() && check_help(values);
```

Comment:
Because if the CLI was passed the following command: `fastddsqosprof file.xml set datareader[profile].qos[] -h` instead of showing the `DATAREADER_QOS_USAGE`, the `DATAREADER_USAGE` was shown. In this case the specific subelement help will be checked within each bifurcation. The `DATAREADER_USAGE` should be shown only if the subelement is not recognized or if the help is given before any valid subelement.